web/src/main/java/com/epam/esm/exceptions/AccessDeniedHandlerEntryPoint.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.esm.exceptions;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.web.access.AccessDeniedHandler;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MimeTypeUtils;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import static com.epam.esm.exceptions.ExceptionCodes.FORBIDDEN_EXCEPTION;
+
+
+@Component
```

Comment:
It would be better if you add javadoc for these new classes. Describe the purpose of these classes in javadoc.